tests/glm/test_distribution.py

```diff
@@ -27,6 +28,7 @@
         (GammaDistribution(), 0),
         (InverseGaussianDistribution(), 0),
         (TweedieDistribution(power=1.5), 0),
+        (NegativeBinomialDistribution(theta=1.0), 0),
```

Comment:
Ideally we should test a value other than 1. theta = 1 leads to a lot of simplifications in the formulas that may hide bugs.